docs/reference/iql/index.md

```diff
@@ -2,12 +2,32 @@
 
 This reference page describes all building blocks of the [Intermediate Query Language (IQL)](../../concepts/iql.md).
 
+
+::: dbally.iql.syntax.Node
+
+
 ::: dbally.iql.syntax.FunctionCall
+    options:
+        members:
+        - name
+        - arguments
 
 ::: dbally.iql.syntax.BoolOp
+    options:
+        members:
+        - match
 
 ::: dbally.iql.syntax.And
+    options:
```

Comment:
Is the "members" option defined everywhere because you want to avoid subclasses having members from their parent classes listed?

I'm not sure about this choice - it's nice to see everything that can be used when using a particular class, no matter where it was originally defined.

And even if you want to hide members from parent classes - we should look for an automatic option. Doing this by hand using a whitelist seems extremely error prone - it almost guarantees that the documentation will become obsolete as new members are added to different classes.

Reply:
It's two-fold:
1. For some classes I want to hide public methods that are used by different classes, but not by the users.
2. The second use case is exactly what you wrote. Personally I prefer parent functions to be hidden if they do not implement different logic and to be shown when they differ in the implementation. But probably this is to be discussed.

I agree that this should be automated will think on how to do this.

Reply:
Yes, so there is an [inherited_members field](https://mkdocstrings.github.io/python/usage/configuration/members/#inherited_members)

Now we have to decide whether we show them or not